ecs-cli/modules/config/config_v1.go

```diff
@@ -45,6 +45,7 @@ type CLIConfig struct {
 	ComposeServiceNamePrefix string
 	ComposeProjectNamePrefix string // Deprecated; remains for backwards compatibility
 	CFNStackName             string
+	CFNStackNamePrefix       string // Removed in new config; remains for backwards compatibility
```

Comment:
For sake of consistency, maybe just say "Deprecated; remains for backwards compatibility" ?